funnel/models/event.py

```diff
@@ -176,6 +199,14 @@ class Attendee(BaseMixin, db.Model):
         backref=db.backref('attendees', cascade='all, delete-orphan', lazy='dynamic'))
     checked_in = db.Column(db.Boolean, default=False, nullable=False)
 
+    @classmethod
+    def get(cls, participant=None, event=None, create=False):
```

Comment:
participant` and `event` shouldn't have default values in this method. Both are required. `event` is the more significant parameter (for SQL left-to-right indexing order) and should be first.